core/scripts/keystone/templates/crib-overrides.yaml

```diff
@@ -0,0 +1,41 @@
+helm:
+  values:
+    chainlink:
+      nodes:
+        - name: node-1
```

Comment:
I think the pattern has been changed here:
https://github.com/smartcontractkit/chainlink/pull/13238/files